### DISCUSSION TITLE
Quest Answer Statistics

### DISCUSSION THREAD

**Discussion by andrewharvey** [streetcomplete/StreetComplete#2673] [Q&A] [answered]
Has anyone compiled stats on quest answers (particularly by region). It would be helpful to see the variance in answers and which quests mostly have the same answer.

**Comment by matkoniecz** [comment]
I did some analysis on number of solved quests, but this kind of analysis would be much more tricker (I was able to check solely changeset metadata)

**Comment by westnordost** [comment, marked as answer]
This analysis would be very hard to do. A script that does this would need to be programmed from scratch. What would be eas(ier) to answer would be the number of quests solved per quest type.
